src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
```

Comment:
Consider moving component styled to the corresponding component's directory for a better project structure, making it easier to find at the same time.